pandas/core/sorting.py

```diff
@@ -374,7 +374,16 @@ def nargsort(
     if not ascending:
         non_nans = non_nans[::-1]
         non_nan_idx = non_nan_idx[::-1]
-    indexer = non_nan_idx[non_nans.argsort(kind=kind)]
+
+    # GH 35922. Move support for object sort here from Series.sort_values
```

Comment:
this comment is not super useful, what is an expl of why you need this try/except e.g. L380 move here

Reply:
No longer relevant, since I removed the try/except.